### ISSUE TITLE
charpress didn't upload the new tag to dockerhub

### ISSUE THREAD

**Issue by rabernat** [pangeo-data/helm-chart#37]
The most recent charpress build succeeded in building a notebook docker image:
https://travis-ci.org/pangeo-data/helm-chart/jobs/391946771#L2019
It tagged it `9d7f08b` but it doesn't appear to have uploaded to dockerhub: https://hub.docker.com/r/pangeo/notebook/tags/.

I expected this, since we don't have any docker credentials stored here. But I also expected an error.

Someone needs to figure out what is going on here. Does this fact that this tag was not uploaded mean that the current helm chart is not deployable? 
https://pangeo-data.github.io/helm-chart/pangeo-v0.1.1-9d7f08b.tgz

(Also, how can I inspect the contents of the helm chart at that link? Do I really have to download it an unzip it or is there an easier tool?)

**Comment by rabernat** [comment]
I just downloaded and unzipped the chart. This is what is in values.yaml:
```
  singleuser:
    image:
      name: pangeo/notebook
      tag: 9d7f08b
```

So the templating system of chartpress did work: it replaced whatever tag was in the commit with the tag from the image it built.

**Comment by rabernat** [comment]
fixed by #40
